### PR TITLE
Adds the "trader" appearance to the list of available agent ID appearances

### LINKED ISSUE #11134: Agent IDs don't have the "Traveler" appearance available
(WEB REPORT BY: wizardcrying REMOTE: 172.93.109.202:7777)

> Revision (Should be above if you're viewing this from ingame!)
> Revision 4fb6a0e943eef40b29944088c8c8e88346dd4616 
> 
> General description of the issue
> The agent ID from the Syndicate Uplink doesn't allow you to change its appearance to that of a "Traveler" ID card.
> 
> What you expected to happen
> To have my Agent ID appear to be a Traveler's ID.
> 
> What actually happened
> There was no option to.
> 
> Steps to reproduce if possible
> 1. Buy Agent ID
> 2. Opt to "Edit" it.
> 3. Select "Appearance".
> 4. Try to find the option and fail.

### PR DESCRIPTION
**Pull request by Wizardcrying**
:100:% tested.
Fuck that list why doesn't it pull from existing ones
Fixes #11134

### CODE REVIEW

File: code/game/objects/items/weapons/cards_ids.dm [hunk 1]
```diff
@@ -351,7 +351,7 @@
 						UpdateName()
 						to_chat(user, "Name changed to [new_name].")
 
-					if("Appearance")
+					if("Appearance") //what a snowflake fuck you
```
Comment (PJB3005) [inline code/game/objects/items/weapons/cards_ids.dm]: Shitposting in the code.

**Comment by Wizardcrying** [comment]
oh wait i need to change the log

**Comment by Wizardcrying** [comment]
You don't need a newline at the end of the .yml, right?

File: code/game/objects/items/weapons/cards_ids.dm (hunk 1 above)
Comment (ComicIronic) [inline code/game/objects/items/weapons/cards_ids.dm]: You're not allowed to complain in the code in a small change. Fix or keep quiet.

Comment (Wizardcrying) [inline code/game/objects/items/weapons/cards_ids.dm]: eh, I might in a future pr
i'll remove the comment because I want this merged for now

**Pull request merged**